server/parsec/components/user.py

```diff
@@ -366,6 +372,10 @@ class UserListFrozenUsersBadOutcome(BadOutcomeEnum):
     AUTHOR_REVOKED = auto()
 
 
+class UserListOrganizationsBadOutcome(BadOutcomeEnum):
```

Comment:
I don't think this is needed: all organization not containing the email are simply omitted from the result, so returning an empty list as a result is enough